packages/flutter/lib/src/material/dialog.dart

```diff
@@ -872,6 +872,11 @@ Widget _buildMaterialDialogTransitions(BuildContext context, Animation<double> a
 /// By default, `useRootNavigator` is `true` and the dialog route created by
 /// this method is pushed to the root navigator.
 ///
+/// The `routeSettings` argument is passed to [showGeneralDialog],
+/// see [RouteSettings] for details.
+///
+/// The `child` argument is deprecated, and should be replaced with `builder`.
```

Comment:
If it's deprecated, then maybe add `@deprecated` to the argument?

Reply:
Looks like it was a weird cherry-pick merge issue. I removed this comment as it wasn't related to the changes and looks like it was already removed from master.